src/managers/CachedManager.js

```diff
@@ -13,6 +14,18 @@ class CachedManager extends DataManager {
 
     Object.defineProperty(this, '_cache', { value: this.client.options.makeCache(this.constructor, this.holds) });
 
+    let cleanup = this._cache[_cleanupSymbol];
+    const cacheType = this._cache.constructor.name;
```

Comment:
This doesn't seem right... `this._cache` is a form of Collection. Did you mean this.Holds instead?

Reply:
The name is probably misleading, when running it I liked `Garbage Collection completed on ThreadManager, which held a SweptCollection`, but open to changes in what this message is.

I suppose it would maybe make sense to say `a SweptCollection of ThreadChannels`?

Reply:
Yeah, personally I'd rather it say what it holds, not what type of collection it is (plus it's always a SweptCollection currently)

`GC completed on ${this.constructor.name}, which had a ${this._cache.constructor.name} of ${this.Holds.name}` or similar